algortihm/Tree.py

```diff
@@ -1,6 +1,6 @@
 import random
 
-from Node import Node
+from algortihm.Node import Node
```

Comment:
if both are moved to the algorithm folder is this required?

Reply:
No, this can be omitted, it works without it even though my vs code shows it as an error. I removed this line.